Backend/Tumultu/Presentation/Program.cs

```diff
@@ -0,0 +1,25 @@
+var builder = WebApplication.CreateBuilder(args);
+
+// Add services to the container.
+
+builder.Services.AddControllers();
+// Learn more about configuring Swagger/OpenAPI at https://aka.ms/aspnetcore/swashbuckle
+builder.Services.AddEndpointsApiExplorer();
+builder.Services.AddSwaggerGen();
```

Comment:
Can you change it to openAPI swash? It will help with migrating models to anular

Reply:
To be honest I've never done that. Will look into it